src/platform/silabs/Logging.cpp

```diff
@@ -15,13 +15,6 @@
 #include <stdio.h>
 #include <string.h>
 
-#ifndef BRD4325A
-
-#ifdef RAIL_ASSERT_DEBUG_STRING
-#include "rail_assert_error_codes.h"
```

Comment:
Isn't that include needed in case of rail assert and getting the correct error message?

Reply:
added back.